assets/4.map

```diff
@@ -5,7 +5,7 @@ XP         |                  A                           PX
 X          |                                               X
 X          |         +-------------------------+           X
 +          |                                   |           +
-|          |   C                               |           |
+|          |   C              E                |           |
 |          |                                   |           |
 |          |                                   |           |
 |          |                                   |           |
```

Comment:
per le mappe non dovresti modificare a mano, c'è lo script in python per farlo, sarebbe da aggiornare, se vuoi faccio io al prossimo PR.

Reply:
https://github.com/EndPass-Game/programmazione/pull/58